### PR TITLE
feat: add ParserWithSendTxs support

### PR DESCRIPTION
**Pull request by Sekhmet**

### CODE REVIEW

File: src/serial_bridge_index.cpp [hunk 1]
```diff
@@ -177,7 +192,7 @@ native_response serial_bridge::extract_data_from_blocks_response(const char *buf
 			bridge_tx.rv = tx.rct_signatures;
 			bridge_tx.pub = get_extra_pub_key(fields);
 			bridge_tx.fee_amount = get_fee(tx, bridge_tx);
-			bridge_tx.inputs = get_inputs(tx, bridge_tx, gki);
+			bridge_tx.inputs = has_send_txs ? get_inputs_with_send_txs(tx, bridge_tx, send_txs) : get_inputs(tx, bridge_tx, gki);
```
Comment (gutenye) [inline src/serial_bridge_index.cpp:195]: Why need `get_inputs_with_send_txs`? 
Use one single `get_inputs` still works.

Comment (Sekhmet) [reply src/serial_bridge_index.cpp:195]: `get_inputs` compares keyImage of each input to globalKeyImages.
`get_inputs_with_send_txs` only checks if txId exists in `sendTxs` and returns all inputs.

**Review by gutenye — APPROVED** [review]
utACK (will test later after build the native module)

**Comment by Sekhmet** [comment]
I will merge it, update `react-native-fast-crypto` and build `@next` npm package.

**Pull request merged**